src/common/constants.ts

```diff
@@ -108,10 +143,31 @@ export function stringToType(str: string): CardType {
 
 // Parsing.
 
-const shouldUseLocalParser = USE_LOCAL_PARSER_ON_LOCALHOST && typeof window !== 'undefined' && window.location.hostname === 'localhost';  // tslint:disable-line no-typeof-undefined
-const liveParserUrl = '//parser.wordbots.io';
-const stagingParserUrl = '//wordbots-parser-staging.herokuapp.com';
-export const PARSER_URL = USE_STAGING_PARSER ? stagingParserUrl : (shouldUseLocalParser ? `http://localhost:${LOCAL_PARSER_PORT}` : liveParserUrl);
+const LIVE_PARSER_URL = '//parser.wordbots.io';
+const STAGING_PARSER_URL = '//parser-staging.wordbots.io';
+const LOCAL_PARSER_URL = 'http://localhost:8080';
+
+export const PARSER_URL: string = (() => {
+  // tslint:disable-next-line: no-typeof-undefined
+  if (typeof window !== 'undefined') {
+    if (['app.wordbots.io', 'wordbots-game.herokuapp.com'].includes(window.location.hostname)) {
+      // On app.wordbots.io or wordbots-game-staging.herokuapp.com, use production DB
+      return LIVE_PARSER_URL;
+    } else if (window.location.hostname === 'localhost') {
```

Comment:
Any reason not to do this as 
```
if (localhost) {
 // local/staging parser URL logic
} else {
  // production
}
```

Reply:
Hmm, it's cause I fall back to staging, so we still have to cover all three cases.

Reply:
Ahh fair enough